rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js

```diff
@@ -427,7 +425,7 @@ function createRuleTable(gridId, data, needCheckbox, needActions, needCompliance
          }
        })
       }
-    , "aaSorting": [[ sortingDefault, "asc" ]]
+    , "aaSorting": [[ 0, "desc" ]]
```

Comment:
That will sort by name in the reverse order :( maybe we should keep asc and reverse our sort order